application/.htaccess

```diff
@@ -1 +1,13 @@
-Deny from all
\ No newline at end of file
+<IfVersion < 2.4>
+    Order allow,deny
+    Deny from all
+</IfVersion>
```

Comment:
Why not

```
<IfVersion < 2.4>
    Order allow,deny
    Deny from all
</IfVersion>
<IfVersion >= 2.4>
        Require all denied
</IfVersion>
```

Reply:
I think, you are partially right. When users are on 2.4+, they can in every case use the `Require` setting.
But if they have additionally enabled `access_compat` they could have set something additional. It's not fully clear which setting is preferred (Apaches documentation says, one should not use require and Allow/Deny together) the compat settings should be overwritten, too.

I will update the pull-request.